android/Hybridge/src/com/pdi/hybridge/HybridgeBroadcaster.java

```diff
@@ -32,11 +32,12 @@ private HybridgeBroadcaster() {
         mJsBuffer = new StringBuffer("");
     }
 
-    public void initJs(WebView view, JSONArray actions, JSONArray events) {
+    public void initJs(WebView view, JSONArray actions, JSONArray events, JSONObject custom) {
```

Comment:
Everything is custom (actions, events...) Maybe would be better a more descriptive name. The same all along the code (JS, Android, iOS).